tests/PHPUnit/Unit/Scheduler/TimetableTest.php

```diff
@@ -61,6 +63,19 @@ public function testGetTimetableFromOptionValue($expectedTimetable, $option)
         self::resetPiwikOption();
     }
 
+    public function testRescheduleTaskAndRunTomorrow()
+    {
+        $timetable = new Timetable();
```

Comment:
Maybe creating a Mock of TimeTable with empty constructor and `save` method could work as well.
It's only those methods requiring the database.